### ISSUE TITLE
UITableViews populated by rx_itemsWithCellIdentifier() are editable

### ISSUE THREAD

**Issue by charlag** [ReactiveX/RxSwift#1005]
**Short description of the issue**:

  When UITableView is populated by rx_itemsWithCellIdentifier() cells can be swapped and "delete" action will be revealed. 

![screen shot 2016-11-17 at 10 11 59](https://cloud.githubusercontent.com/assets/3099142/20379640/c94e326a-acb6-11e6-8996-9566377d4b1d.png)

**Expected outcome**:

  UITableView is not editable

**What actually happens**:

  UITableView is editable

**Self contained code example that reproduces the issue**:

```swift
tableView.registerClass(UITableViewCell.self, forCellReuseIdentifier: "detailCell")

Observable.just(["1", "2", "3"])
         .bindTo(tableView.rx_itemsWithCellIdentifier("detailCell",
               cellType: UITableViewCell.self)) { row, value cell in
                    cell.textLabel?.text = value
         }.addDisposableTo(disposeBag)
```

**RxSwift/RxCocoa/RxBlocking/RxTest version/commit**

  2.6.0

**Platform/Environment**

  - [x] iOS
  - [ ] macOS
  - [ ] tvOS
  - [ ] watchOS
  - [ ] playgrounds

**How easy is to reproduce? (chances of successful reproduce after running the self contained code)**

  - [x] easy, 100% repro
  - [ ] sometimes, 10%-100%
  - [ ] hard, 2% - 10%
  - [ ] extremely hard, %0 - 2%

**Xcode version**:

```
  8.1
```

**Installation method**:
  - [x] CocoaPods
  - [ ] Carthage
  - [ ] Git submodules

**I have multiple versions of Xcode installed**:
  (so we can know if this is a potential cause of your issue)
  - [ ] yes (which ones)
  - [x] no

**Level of RxSwift knowledge**:
  (this is so we can understand your level of knowledge
    and formulate the response in an appropriate manner)
  - [ ] just starting
  - [ ] I have a small code base
  - [x] I have a significant code base

**Comment by charlag** [comment]
As far as I understand it is fixed in the current version: #907
Any chance it will be backported?

**Comment by kzaher** [comment]
Hi @charlag ,

the solution how to change that default behavior for Swift 2.0 branch is

```swift
   // viewDidLoad
    tableView.rx
            .setDelegate(self)
            .addDisposableTo(disposeBag)

// to prevent swipe to delete behavior
func tableView(_ tableView: UITableView, editingStyleForRowAt indexPath: IndexPath) -> UITableViewCellEditingStyle {
    return .none
}
```

We aren't planning to change this behavior for 2.x branch since that would be a breaking change.
